<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationSelectionDialog</class>
 <widget class="QDialog" name="StationSelectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Station Selection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="buttonLayout">
       <item>
        <widget class="QPushButton" name="selectionOK">
         <property name="text">
          <string>Close</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="selectAllStations">
         <property name="text">
          <string>Choose &amp;all stations</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="buttonSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="StationTable" name="stationTable" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StationTable</class>
   <extends>QWidget</extends>
   <header>StationSelection.hh</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
